src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1524,7 +1524,7 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
     }
 
     void visit_ListItem(const ASR::ListItem_t& x) {
-        ASR::ttype_t* el_type = ASRUtils::get_contained_type(
+        ASR::ttype_t *el_type = ASRUtils::get_contained_type(
```

Comment:
Please revert the style changes.

Reply:
Thank you very much for bringing this to notice @Shaikh-Ubaid! We really do not need these changes as there are no real changes. :+1: I forgot to revert them when reverting the previous changes. I am doing this now.